pkg/model/api/v1/secret.go

```diff
@@ -58,9 +58,6 @@ func (s *SecretSpec) UnmarshalYAML(unmarshal func(interface{}) error) error {
 }
 
 func (s *SecretSpec) validate() error {
-	if s.BasicAuth == nil && s.Authorization == nil {
-		return fmt.Errorf("at most one of basicAuth and authorization must be configured")
-	}
 	if s.BasicAuth != nil && s.Authorization != nil {
```

Comment:
Can you modify this condition, oauth is missing in this check.

```suggestion
if s.BasicAuth != nil && s.Authorization != nil && s.OAuth != nil {
}
```